invenio_app_ils/records_relations/views.py

```diff
@@ -121,19 +129,20 @@ def _delete_parent_child_relation(self, record, relation_type, payload):
         Expected payload:
 
             {
-                parent_pid: <pid_value>,
+                parent_pid_value: <pid_value>,
                 parent_pid_type: <pid_type>,
-                child_pid: <pid_value>,
+                child_pid_value: <pid_value>,
                 child_pid_type: <pid_type>,
                 relation_type: "<Relation name>"
             }
         """
-        parent_pid, parent_pid_type, child_pid, child_pid_type, _ = \
-            self._validate_parent_child_creation_payload(payload)
+        parent_pid_value, parent_pid_type, child_pid_value, child_pid_type, _ = self._validate_parent_child_creation_payload(
```

Comment:
pep will complain about this one

Reply:
I thought the same, but it does not and `black` is re-formatting like this. No idea...